src/gui/guiFormSpecMenu.cpp

```diff
@@ -2580,14 +2583,21 @@ void GUIFormSpecMenu::parsePadding(parserData *data, const std::string &element)
 			<< "'" << std::endl;
 }
 
-bool GUIFormSpecMenu::parseStyle(parserData *data, const std::string &element, bool style_type)
+void GUIFormSpecMenu::parseStyle(parserData *data, const std::string &element)
 {
+	if (data->parser_type != "data->parser_type" && data->parser_type != "style_type") {
+		errorstream << "Invalid style element type: '" << data->parser_type << "'" << std::endl;
+		return;
+	}
+
+	bool style_type = (data->parser_type == "style_type");
+
 	std::vector<std::string> parts = split(element, ';');
 
 	if (parts.size() < 2) {
 		errorstream << "Invalid style element (" << parts.size() << "): '" << element
 					<< "'" << std::endl;
-		return false;
+		return;
 	}
```

Comment:
`/test_formspec` in devtest spams the error `ERROR[Main]: Invalid style element type: 'style'`

Reply:
ok i'll check this day

Reply:
fixed my bad